### PR TITLE
Fix argon2 in php 7.4 and above 

### PR DESCRIPTION
**Pull request by arroyc**
- [X] The purpose of this PR is explained in this message or in an issue. If an issue please include a reference as \#\<issue\_number\>.
- [X] Tests are included and/or updated for code changes.
- [X] Proper license headers are included in each file.

### CODE REVIEW

File: doc/supportedPlatformVerions.md [hunk 1]
```diff
@@ -1,67 +1,22 @@
 # Supported platforms and versions
 
-## Supported stretch versions
+## dotnet
 
-### dotnet
```
Comment (qianz2) [inline doc/supportedPlatformVerions.md:5]: we still want to keep these versions for documentation, could you revert the changes in this file

**Review by qianz2 — APPROVED** [review]

File: doc/supportedPlatformVerions.md (hunk 1 above)
Comment (arroyc) [reply doc/supportedPlatformVerions.md:5]: yeah thanks .. i will undo this change
